google/cloud/storage/internal/patch_builder_details.h

```diff
@@ -0,0 +1,48 @@
+// Copyright 2018 Google LLC
```

Comment:
This copyright year may need to be updated since this is a brand new file.

Reply:
Ooops, fixed.